grails-forge-core/src/main/java/org/grails/forge/feature/build/gradle/templates/dependency.rocker.raw

```diff
@@ -1,3 +1,3 @@
 @args(String groupId, String artifactId, String scope, String version, boolean pom)
 
-@(pom ? scope + " platform" : scope)("@groupId:@artifactId@(version != null ? ":" + version : "")")
+@(pom ? scope + " platform" : scope) "@groupId:@artifactId@(version != null ? ":" + version : "")"
```

Comment:
I tried to add the `grails-bom` to the `base` feature and the parenthesis did come out ok.
However, there was a spacing issues: `implementation platform ( "org.grails:grails-bom:6.2.2")`

Reply:
The extra spaces where coming from https://github.com/grails/grails-forge/pull/424/files#diff-ae814cab294ecd3c3a3fdfd4f7df62fbdaa5653fac1f6f0451f267265c7395ee, which is now fixed.

But, I found that the above code, which is doing the same thing, is only used for 3 dependencies where the Gradle scope has not been implemented in the rest of grails-forge.  I am going to add the missing scopes (`testImplementation` and `testRuntimeOnly`), move those over to remove dependency.rocker.raw, since toSnippet() is all we will need. 
https://github.com/grails/grails-forge/blob/6.2.x/grails-forge-core/src/main/java/org/grails/forge/feature/build/gradle/templates/dependencies.rocker.raw#L20

Reply:
https://github.com/grails/grails-forge/pull/424/commits/3647ba8eb162a7ec692a59d84337f47a503e070d fully implements `testImplementation` and `testRuntimeOnly` which were there but needed clearer naming.